cli/src/semgrep/target_manager.py

```diff
@@ -471,15 +477,15 @@ def files_from_filesystem(self) -> FrozenSet[Path]:
         )
 
     @lru_cache(maxsize=None)
-    def files(self) -> FrozenSet[Path]:
+    def files(self, ignore_baseline_handler: bool = False) -> FrozenSet[Path]:
```

Comment:
probably want to edit function level comment on what ignore_baseline_handler does when set to true (maybe also why we need it)